### PR TITLE
Dev ltclm swisssearch swissnames3d

### PR DESCRIPTION
**Pull request by ltclm**
improve swissnames label,
labels are pregenerated while data import now

### CODE REVIEW

**Comment by ltclm** [comment]
gazetteer label is not truncated anymore, as discussed this will be done in map.geo.admin.ch client code.

File: conf/search.conf.part [hunk 1]
```diff
@@ -96,12 +96,12 @@ source src_swissnames3d : src_swisssearch
         , st_y(st_transform(st_geometryn(the_geom,1),4326)) as lat \
         , st_x(st_transform(st_geometryn(the_geom,1),4326)) as lon \
         , 1 as num \
-        FROM tlm.swissnames3d_point \
+        FROM tlm.swissnames3d_point f\
```
Comment (gjn) [inline conf/search.conf.part]: What's the `f` doing?

**Comment by ltclm** [comment]
the f was just a table alias which is not used anymore, thanks for the remark.
the sql query should be clean now.

**Comment by gjn** [comment]
Are the dev indices up-to-date?

**Comment by ltclm** [comment]
yes, i have just updated the indices.

**Pull request merged**